src/coreclr/pal/src/libunwind/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@
 # It overwrites the one found in upstream
 project(unwind)
 
+# Disable C11 compat
+unset(CMAKE_C_STANDARD)
+unset(CMAKE_C_STANDARD_REQUIRED)
```

Comment:
libunwind is a C library. I think unsetting `CMAKE_CXX_` in `if(CLR_CMAKE_HOST_WIN32)` block (at line 100 below) be more a accurate fix. cc @janvorli

Reply:
libunwind is a C library. I think setting `project(unwind C)` on line 3 would be more a accurate fix. cc @janvorli